src/main/java/com/spikes2212/command/genericsubsystem/commands/RiseGenericSubsystem.java

```diff
@@ -0,0 +1,89 @@
+package com.spikes2212.command.genericsubsystem.commands;
+
+import java.util.function.Supplier;
+
+import com.spikes2212.command.genericsubsystem.GenericSubsystem;
+
+import edu.wpi.first.wpilibj.Timer;
+
+/**
+ *
+ * This command rise the speed of a {@link GenericSubsystem} linearly so it will
+ * reach a wanted speed after a given time.
+ *
+ * @author Yuval Levy
+ *
+ */
+public class RiseGenericSubsystem extends MoveGenericSubsystem {
```

Comment:
we can rename this command to `AccelerateSubsystem` or `AccelerateGenericSubsystem`. this command should just make a subsystem reach a sertain speed. we can then hook it up with other command using the `andThen` method from command.